### ISSUE TITLE
Please tag releases

### ISSUE THREAD

**Issue by cowwoc** [slightlyoff/Promises#56]
Hi,

I'd like to publish this Javascript file using http://www.webjars.org/. In order for users to reference your library, I need to introduce some sort of version number or tag.

Is it possible for you to tag future releases?

**Comment by slightlyoff** [comment]
Once I rename Future to Promise globally, I'm happy to do that. Look for that later today.

**Comment by fbender** [comment]
How's this going along?

Would be helpful to name tags after the WHATWG spec date the code matches, e. g. if your polyfill matches the spec from today, the tag should be 20130620 (or similar). That way, one can easily track deviations from specific revisions.

**Comment by cowwoc** [comment]
Good idea though you'll need to be able to increment the version for bug
fixes even if the spec remains the same.
